posts/2024/2024-06-10_help_forum_integration.md

```diff
@@ -0,0 +1,20 @@
+---
+media:
+  - mastodon-eu-freiburg
+
+hashtags:
+  mastodon:
```

Comment:
This line should also be the same as media name `mastodon-eu-freiburg`

Reply:
please add a suggestion @arash77 https://haacked.com/archive/2019/06/03/suggested-changes/

This way its super easy to accept it and you get credits for it.